packages/orchestration/src/exos/ica-account-kit.js

```diff
@@ -8,22 +8,31 @@ import {
   CosmosChainAddressShape,
   OutboundConnectionHandlerI,
   Proto3Shape,
-  TxBodyOptsShape,
+  ExecuteICATxOptsShape,
 } from '../typeGuards.js';
 import { findAddressField } from '../utils/address.js';
 import { makeTxPacket, parseTxPacket } from '../utils/packet.js';
 
 /**
  * @import {HostOf} from '@agoric/async-flow';
  * @import {Zone} from '@agoric/base-zone';
- * @import {Connection, Port} from '@agoric/network';
- * @import {Remote, Vow, VowTools} from '@agoric/vow';
+ * @import {Bytes, Connection, Port} from '@agoric/network';
+ * @import {Remote, Vow, PromiseVow, VowTools} from '@agoric/vow';
  * @import {AnyJson} from '@agoric/cosmic-proto';
  * @import {TxBody} from '@agoric/cosmic-proto/cosmos/tx/v1beta1/tx.js';
  * @import {LocalIbcAddress, RemoteIbcAddress} from '@agoric/vats/tools/ibc-utils.js';
  * @import {CosmosChainAddress, IcaAccount} from '../types.js';
  */
 
+/**
+ * @typedef {Omit<Connection, 'send'> & {
+ *   send: (
+ *     packetBytes: Bytes,
+ *     opts?: { relativeTimeoutNs?: bigint },
+ *   ) => PromiseVow<Bytes>;
+ * }} IBCConnection
+ */
```

Comment:
I considered specifying this somewhere in `@agoric/vats`, but couldn't find the right place. Open to suggestions.

Reply:
I'd remove this type definition, and instead change the definition of `import('@agoric/network').Connection['send']` to have a second argument of `opts?: SendOptions`.  Since the option properties shouldn't conflict between different network protocols, SendOptions doesn't need to be IBC-specific, it should just declare `relativeTimeoutNs?: bigint` as a part of the Network API.